### DISCUSSION TITLE
NGINX Proxy

### DISCUSSION THREAD

**Discussion by mtnbiker777** [photoprism/photoprism#1074] [General]
I've looked through the documentation and done what's been suggested but I can't seem to get NGINX reverse proxy working. I'm trying to proxy it as a subdirectory. My config looks like this:

location /photoprism/ {
      proxy_pass http://172.18.0.27:2342/;
  }

Seems simple enough but it doesn't work, all I get is this:
![proxy](https://user-images.githubusercontent.com/39917149/108395619-fe20a800-71e3-11eb-8751-649242e1ced3.jpg)

**Comment by lastzero** [comment]
See https://docs.photoprism.org/getting-started/faq/#can-i-deploy-photoprism-on-a-shared-domain-like-httpsexamplecomphotoprism

**Reply by mtnbiker777** [reply]
Right, ok thanks, saves me wasting time trying to make it work.

**Reply by lastzero** [reply]
Sorry we have no other news :)
